package.json

```
@@ -1,45 +1,52 @@
 {
   "name": "@thesharks/jagtag-js",
-  "version": "1.1.0",
+  "version": "2.0.0",
   "description": "A JavaScript port of https://github.com/jagrosh/JagTag.",
-  "main": "index.js",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "files": [
+    "dist"
+  ],
   "scripts": {
-    "start": "cd website && npm start",
-    "install-docs": "deps.bat",
-    "start-docs": "mkdocs serve",
-    "build": "mkdocs build",
-    "test": "eslint . && jest --config jest.coverage.js && codecov --disable=gcov",
-    "test-local": "eslint . && jest --config jest.coverage.js",
-    "test-coverage": "jest --watch --config jest.coverage.js",
-    "test-dev": "jest --watch"
+    "build": "tsc",
+    "prepublishOnly": "npm run build",
+    "test": "jest",
+    "test:lint": "eslint .",
+    "test:coverage": "jest --config jest.coverage.js",
+    "test:coverage:watch": "jest --watch --config jest.coverage.js",
```

There is apparently (as confirmed in my review process of this PR) a bug in Jest where using `--watch` in conjunction with coverage causes this weird issue where coverage only reports for the entrypoint. This is a known issue; see https://github.com/facebook/jest/issues/7331 and https://github.com/facebook/jest/issues/7331.

This can be resolved by using `--watchAll` as suggested by https://github.com/facebook/jest/issues/7331#issuecomment-511121019.

```suggestion
    "test:coverage:watch": "jest --watchAll --config jest.coverage.js",
```